packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -131,19 +147,19 @@ export function SquadPageHeader({
       {squad.description && (
         <p
           className={classNames(
-            'mt-6 w-full text-center text-text-tertiary typo-body',
+            'mt-5 w-full text-center text-text-secondary typo-callout',
             !shouldUseListMode && 'laptopL:text-left',
             !shouldUseListMode && MAX_WIDTH,
           )}
         >
           {squad.description}
         </p>
       )}
-      <SquadHeaderBar squad={squad} members={members} className="mt-8" />
-      <span className="mt-6 text-text-quaternary typo-footnote">
+      <SquadHeaderBar squad={squad} members={members} className="mt-5" />
+      <span className="mt-6 font-bold text-text-tertiary typo-caption1">
```

Comment:
Could move these to Typography, but optional :)

Reply:
Done!